app/Http/Controllers/Home/HomeController.php

```diff
@@ -1,42 +1,42 @@
 <?php
 
-namespace App\Http\Controllers\Chat;
+namespace App\Http\Controllers\Home;
 
 use App\Http\Controllers\Controller;
 use Illuminate\Auth\Access\Response;
 use App\Settings\PathInfo;
 
-class ChatController extends Controller
+class HomeController extends Controller
 {
     /*
     |--------------------------------------------------------------------------
-    | ChatController
+    | HomeController
     |--------------------------------------------------------------------------
     |
-    | チャット機能が実装されているコントローラ
+    | Homeコントローラ
     |
     */
 
     /**
-     * Create a new authentication controller instance.
+     * Create a new home controller instance.
      *
      * @return void
      */
     public function __construct()
     {
-        $pathInfo = new PathInfo();
-        $pathInfo->getNodePath();
+        // $pathInfo = new PathInfo();
+        // $pathInfo->getNodePath();
 
     }
 
     /**
-     * indexを開く画面
+     * topアクション
      *
```

Comment:
__[code_sniffer]__
* The closing brace for the class must go on the next line after the body